src/controllers/__tests__/item.test.js

```diff
@@ -94,12 +94,40 @@ const noDescriptionItem = {
   type: 'Stationery',
 };
 
+const mockUser = {
+  username: 'chandler.bing',
+  firstName: 'Chandler',
+  lastName: 'Bing',
+  email: 'chandlerbing@gmail.com',
+  password: 'password1234',
+  passwordConfirm: 'password1234',
+  address: 'Central Perk, New York',
+  acceptTerms: true,
+};
+
 describe('Items Endpoints', () => {
   beforeAll(async () => {
     await clearDatabase();
+    const response = await request(server)
+      .post('/api/auth/signup')
+      .set('Content-Type', 'application/json')
+      .send(mockUser);
+    /* eslint-disable no-underscore-dangle */
+    ownerId = response.body._id;
+    trueItem.owner = ownerId;
+    trueItem2.owner = ownerId;
+    trueItem3.owner = ownerId;
+    trueItem4.owner = ownerId;
+    const signInResponse = await request(server)
+      .post(`/api/auth/signin`)
+      .set('Content-Type', 'application/json')
+      .send({ username: mockUser.username, password: mockUser.password });
+
+    [authCookie] = signInResponse.headers['set-cookie'];
```

Comment:
Why is `authCookie` wrapped in `[]`?

Reply:
eslint didn't allow me to do `signInResponse.headers['set-cookie'].token`, it suggested to use array destructuring